### ISSUE TITLE
Crash with ClassCastException

### ISSUE THREAD

**Issue by mvarnagiris** [BelooS/ChipsLayoutManager#17]
To reproduce:
1. In device developer options turn on `Don't keep activities`.
2. Open activity with ChipsLayoutManager.
3. Press device home button.
4. Open activity with ChipsLayoutManager again.

This happens when you leave the app in the background for a while and activity is killed by the system.

If you cannot reproduce like that, let me know, I'll provide more detailed setup.

```
Fatal Exception: java.lang.RuntimeException: Unable to start activity ComponentInfo{com.emoticast.tunemoji.android.debug/com.emoticast.tunemoji.feature.main.MainActivity}: java.lang.ClassCastException: android.support.v7.widget.LinearLayoutManager$SavedState cannot be cast to com.beloo.widget.chipslayoutmanager.ParcelableContainer
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2665)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2726)
       at android.app.ActivityThread.-wrap12(ActivityThread.java)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1477)
       at android.os.Handler.dispatchMessage(Handler.java:102)
       at android.os.Looper.loop(Looper.java:154)
       at android.app.ActivityThread.main(ActivityThread.java:6119)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:886)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:776)
Caused by java.lang.ClassCastException: android.support.v7.widget.LinearLayoutManager$SavedState cannot be cast to com.beloo.widget.chipslayoutmanager.ParcelableContainer
       at com.beloo.widget.chipslayoutmanager.ChipsLayoutManager.onRestoreInstanceState(ChipsLayoutManager.java:413)
       at android.support.v7.widget.RecyclerView.onRestoreInstanceState(RecyclerView.java:1188)
       at android.view.View.dispatchRestoreInstanceState(View.java:15751)
       at android.view.ViewGroup.dispatchThawSelfOnly(ViewGroup.java:3251)
       at android.support.v7.widget.RecyclerView.dispatchRestoreInstanceState(RecyclerView.java:1205)
       at android.view.ViewGroup.dispatchRestoreInstanceState(ViewGroup.java:3237)
       at android.view.ViewGroup.dispatchRestoreInstanceState(ViewGroup.java:3237)
       at android.view.ViewGroup.dispatchRestoreInstanceState(ViewGroup.java:3237)
       at android.view.ViewGroup.dispatchRestoreInstanceState(ViewGroup.java:3237)
       at android.view.ViewGroup.dispatchRestoreInstanceState(ViewGroup.java:3237)
       at android.view.ViewGroup.dispatchRestoreInstanceState(ViewGroup.java:3237)
       at android.view.View.restoreHierarchyState(View.java:15729)
       at com.android.internal.policy.PhoneWindow.restoreHierarchyState(PhoneWindow.java:2106)
       at android.app.Activity.onRestoreInstanceState(Activity.java:1047)
       at android.app.Activity.performRestoreInstanceState(Activity.java:1002)
       at android.app.Instrumentation.callActivityOnRestoreInstanceState(Instrumentation.java:1174)
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2638)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2726)
       at android.app.ActivityThread.-wrap12(ActivityThread.java)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1477)
       at android.os.Handler.dispatchMessage(Handler.java:102)
       at android.os.Looper.loop(Looper.java:154)
       at android.app.ActivityThread.main(ActivityThread.java:6119)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:886)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:776)
```

**Comment by BelooS** [comment]
Silly mistake with storing/restoring lm state. Have fixed, would be in the next release

**Comment by mvarnagiris** [comment]
Nice! Thanks again for quick response and resolution.

**Comment by BelooS** [comment]
fixed in 0.3.3
